### ISSUE TITLE
"Copy to subsequent media" does not work if depicts Wikidata place

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#5057]
Steps to reproduce:

1. Take several pictures near a Wikidata place
2. Upload both together.
3. When asked "Is this a picture of [...]", click "Yes".
4. Write a caption/description
5. Tap "Copy to the next items"
6. Tap "Next"
7. Expected: The caption/description from step 4 has not been copied. Problem: The caption/description is from the Wikidata place.

Workaround: Tap "Previous" and tap "Copy to subsequent media" again. This time it works.

**Comment by nicolas-raoul** [comment]
Interesting, there may be special conditions I have not thought of.

**Comment by junhaow1** [comment]
Hi @nicolas-raoul  Could I please work on this issue? Pls assign me. Thank you:)

**Comment by nicolas-raoul** [comment]
@junhaow1 Yes thanks! First, please let us know whether you are able to reproduce the issue.

**Comment by arinmodi** [comment]
Can I work on these issue if @junhaow1 not working ?

**Comment by nicolas-raoul** [comment]
@arinmodi Feel free to work on it, thanks!

**Comment by arinmodi** [comment]
@nicolas-raoul, not able to reproduce

**Comment by ayushanand308** [comment]
@nicolas-raoul As suggested by you, I will be trying to work on this issue first :)

**Comment by ayushanand308** [comment]
@nicolas-raoul 
When I reproduce the issue, I am not getting any bug occurrence. The captions and description is getting copied to the whole batch without having tapped the previous. Am I missing something here?

**Comment by ayushanand308** [comment]
When setting the development environment, I am getting the following errors

![Screenshot 2023-02-27 164022](https://user-images.githubusercontent.com/99035102/221565137-29935bfe-8b37-4790-89f0-e4956a532009.png)

**Comment by nicolas-raoul** [comment]
@ayushanand308 For such errors I recommend searching the web for these error messages.

**Comment by nicolas-raoul** [comment]
@arinmodi @ayushanand308 I just tried and was able to reproduce the issue the first time. Make sure you are getting the popup at step 3. If not, edit the pictures' EXIF to be at the location of the Eiffel Tower, then select both and share to the Commons app.

**Comment by ayushanand308** [comment]
@nicolas-raoul  I followed what you said and was able to successfully reproduce the issue.

**Comment by nicolas-raoul** [comment]
@ayushanand308 Great! Do you want to be assigned to this issue? 🙂

**Comment by ayushanand308** [comment]
@nicolas-raoul Yes!

**Comment by nicolas-raoul** [comment]
@ayushanand308 It is yours! Please let us know about your progress every few days. Thanks! :-)

**Comment by ayushanand308** [comment]
Is this how it is actually supposed to work?
https://user-images.githubusercontent.com/99035102/222404635-dbdeb5fb-677d-4904-9c4a-88ff6fd316df.mp4

**Comment by nicolas-raoul** [comment]
Yes exactly 🙂

**Comment by ayushanand308** [comment]
@nicolas-raoul  I have made a PR. Can you kindly review it. :-).

**Comment by nicolas-raoul** [comment]
@ayushanand308 Did you get a chance to look at my pull request feedback at https://github.com/commons-app/apps-android-commons/pull/5163#issuecomment-1464584775 ? Thanks! If you have no time, we can unassign you and someone else will build upon your pull request, no problem. :-)

**Comment by samimshoaib01** [comment]
Hi , @nicolas-raoul  can i work on this issue ?

**Comment by nicolas-raoul** [comment]
It is yours, thanks! 🙂
Are you able to reproduce the issue?

**Comment by nicolas-raoul** [comment]
This is still happening.
Anyone willing to work on it? First please make sure you are able to reproduce the issue, it is quite easy to do.
Thanks! 🙂

https://github.com/user-attachments/assets/57d564e1-80ee-451e-8a93-ee37877b710a
